templates/active_job.rb

```diff
@@ -1,4 +1,5 @@
 require "active_job/logging"
+require "active_job/log_subscriber"
```

Comment:
Is this also necessary for Rails 6.1?

Reply:
Oddly enough, yes. Without it, we get `uninitialized constant ActiveJob::Logging::LogSubscriber`.